### PR TITLE
added installation via Powershell Gallery

### PR DESCRIPTION
**Pull request by bottkars**
this is for Powershell Users, utilizing PSGallery as default Repo for installing kubectl on Windows

image
image

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6086)

### CODE REVIEW

**Comment by bottkars** [comment]
cla requested 11 Minutes Ago.

**Comment by steveperry-53** [comment]
@bottkars Can you check the **Allow edits from maintainers** box. It should be in the right column. If it's already checked, uncheck and recheck it. Thanks.

File: docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -112,6 +112,19 @@ kubectl is available as a [snap](https://snapcraft.io/) application.
 
 2. Run `kubectl version` to verify that the version you've installed is sufficiently up-to-date.
 
+## Install with Powershell from PSGallery
+
+1. If you are on Windows and using [Powershell Gallery](https://https://www.powershellgallery.com/) package manager, you can install and update with:
```
Comment (heckj) [inline docs/tasks/tools/install-kubectl.md]: you've got a duplicate `https://` for the Powershell Gallery link...

File: docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -112,6 +112,19 @@ kubectl is available as a [snap](https://snapcraft.io/) application.
...
+
+       Install-Script -Name install-kubectl -Scope CurrentUser -Force     
+       install-kubectl.ps1 [-DownloadLocation <path>]
+       
+If no Downloadlocation is specified, kubectl will be installed in users temp Directory   
+2. The installer creates $HOME/.kube and instucts to create a config file
```
Comment (heckj) [inline docs/tasks/tools/install-kubectl.md]: Slightly better wording might be:
> The installer creates `$HOME/.kube` and instucts you to create a config file

File: docs/tasks/tools/install-kubectl.md [hunk 3]
```diff
@@ -112,6 +112,19 @@ kubectl is available as a [snap](https://snapcraft.io/) application.
...
+## Install with Powershell from PSGallery
+
+1. If you are on Windows and using [Powershell Gallery](https://https://www.powershellgallery.com/) package manager, you can install and update with:
+
+       Install-Script -Name install-kubectl -Scope CurrentUser -Force     
+       install-kubectl.ps1 [-DownloadLocation <path>]
+       
+If no Downloadlocation is specified, kubectl will be installed in users temp Directory   
+2. The installer creates $HOME/.kube and instucts to create a config file  
+3. Updating
```
Comment (heckj) [inline docs/tasks/tools/install-kubectl.md]: Can you structure this like the example commands in 1. above - using some wording to explain is before it. For example:

You can update the version of kubectl by rerunning the same scripts again:

    Install-Script -Name install-kubectl -Scope CurrentUser -Force     
    install-kubectl.ps1 [-DownloadLocation <path>]

**Comment by heckj** [comment]
Thanks @bottkars 
/lgtm
/hold cancel

**Comment by darkvn79** [comment]
has problems with setting because , i think they in my system now ,  hackers

user-192
user-48
user-40
user-32
user
guest

**Pull request merged**